rust-cheat-sheet/rust-cheat-sheet/code/01_function_variables.rs

```diff
@@ -0,0 +1,20 @@
+// Declaring a function with type annotated arguments
+// and a specified return type.
```

Comment:
I feel like this implies you can have functions without type annotated arguments.